R/variable.R

```diff
@@ -74,6 +74,20 @@ setMethod("digits<-", "CrunchVariable", function (x, value) {
     halt("digit specifications can only be set for numeric variables")
 })
 
+
```

Comment:
These should probably go up with the other CrunchVariable methods, and notice with those how you can just do `x$description` etc. because of how `$` is defined for tuples.

And since you're doing that, and since the CrunchVariable methods are all like `tuple(object)$alias`, you could make those be `alias(tuple(object))` and pass through the same method.

Reply:
Moved and changed the execution path to all go through the VariableTuple for `description`, `alias`, `name`, and `notes`